src/js/components/jtable/TableRow.jsx

```diff
@@ -13,6 +14,7 @@ type Props = {
     className?: string,
     children?: ReactChildren,
     href?: string,
+    linkTo?: string,
```

Comment:
It's a devDependency, but it probably should be peer. But `<TabLink/>` already uses it, and I don't want to push that complexity into n instances of client code, when it can be in this library. IMO That's what libraries are for.